test/basic.js

```diff
@@ -7,27 +7,23 @@ var tap = require('tap')
 var globalBefore = Object.keys(global)
 var mm = require('../')
 var patterns = require('./patterns.js')
-var regexps = patterns.regexps
-var re = 0
 
 tap.test('basic tests', function (t) {
   var start = Date.now()
 
-  // [ pattern, [matches], MM opts, files, TAP opts]
   patterns.forEach(function (c) {
-    if (typeof c === 'function') return c()
     if (typeof c === 'string') return t.comment(c)
 
-    var pattern = c[0],
```

Comment:
The original code passed `standard`, why do you think changes like this are necessary?